site/gatsby-site/src/components/RelatedIncidents.js

```diff
@@ -55,11 +56,11 @@ const reportsWithIncidentIds = async (reports, client) => {
   }));
 };
 
-const searchColumns = {
+const allSearchColumns = {
   byDatePublished: {
     header: (incident) => (
       <>
-        Incidents reports matched by published date: <b>{incident.date_published}</b>
+        <Trans>Incidents reports matched by published date:</Trans> <b>{incident.date_published}</b>
```

Comment:
Informes de incidentes por fecha de publicación:

(this should be only one trans component that includes the `b` tags)